Sources/InternxtSwiftCore/Utils/CryptoUtils.swift

```diff
@@ -60,5 +60,24 @@ public struct CryptoUtils {
             return nil
         }
     }
+    
+    public func validate(mnemonic: String) -> Bool {
+        let normalizedMnemonic = mnemonic.lowercased().trimmingCharacters(in: .whitespacesAndNewlines)
+        let mnemonicComponents = normalizedMnemonic.components(separatedBy: " ")
+        guard !mnemonicComponents.isEmpty else {
+          return false
+        }
+
+        if String.englishMnemonics.contains(mnemonicComponents[0]) {
```

Comment:
I will use a map rather than a string (convert it dynamically, p.e) so the access time for each word is O(1) (constant). Therefore, this operation does not affect the performance / time-to-login of the app.